### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by aaaaaaaaaaaaaaaaadd22** [andreped/super-ml-pets#42]
What is the recommended setting for training, and how long have you been training the v6 one?

**Comment by andreped** [comment]
Default settings are quite good. I have tuned them myself. However, you could try to increase batch size to 1024 and play around with the learning rate. I have not spent that much time on it yet.
